tests/helpers.py

```diff
@@ -0,0 +1,10 @@
+from unittest.mock import MagicMock
+
+
+__all__ = ('AsyncMock',)
+
+
+# TODO: Remove me on 3.8
+class AsyncMock(MagicMock):
```

Comment:
Can you add a comment explaining what this is or why it is needed? I have no idea.

Reply:
Added